dynamic-layers/recipes-bsp/u-boot/u-boot-variscite/0003-HMX-custom-imx8mp_var_dart.patch

```diff
@@ -24,12 +23,16 @@ V3.
 Add set product-number to device tree on boot.
 Add set rbg led controller register to device tree on boot.
 Add load support for m7 co cpu (hmx-m7-fw.bin), the m7 will load before boot linux and flashing bsp.
-Subject: [PATCH 1/1] patch
+
+v4.
+Change name of the cocpu binary
+Add ddr memory load for cocpu
+Change order of loadning cocpu firmware, now after flashing so it does not timeout and reset the system before it has finished flashing.
```

Comment:
```suggestion
v4.
Change name of the cocpu binary
Add DDR memory load for cocpu
Delay loading the co-cpu firmware until after flashing so that it doesn't timeout and reset the system before it has finished flashing.
```